src/blocks/modal.css

```diff
@@ -76,6 +114,11 @@
   padding-bottom: 10.26px;
 }
 
```

Comment:
Needs correction: the disabled submit button should be grey and not interactive (the cursor of the disabled button should not be `pointer`)